src/development/LocalTest/Services/LocalApp/Implementation/LocalAppHttp.cs

```diff
@@ -79,5 +88,52 @@ public LocalAppHttp(IHttpClientFactory httpClientFactory, IOptions<LocalPlatform
 
             return ret;
         }
+
+        public async Task<Instance?> Instantiate(string appId, Instance instance, string xmlPrefill, string xmlDataId)
+        {
+            var requestUri = $"{appId}/instances";
+            var serializedInstance = JsonSerializer.Serialize(instance, new JsonSerializerOptions { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault });
+
+            var content = new MultipartFormDataContent();
+            content.Add(new StringContent(serializedInstance, System.Text.Encoding.UTF8, "application/json"), "instance");
+            if (!string.IsNullOrWhiteSpace(xmlPrefill) && xmlDataId is not null)
+            {
+                content.Add(new StringContent(xmlPrefill, System.Text.Encoding.UTF8, "application/xml"), xmlDataId);
+            }
+
+            using var message = new HttpRequestMessage(HttpMethod.Post, requestUri);
+            message.Content = content;
+            // TODO: Figure out how to get orgnumber for app owner from appId
+            message.Headers.Authorization = new ("Bearer", GetOrgToken(appId.Split("/")[0], "840747972"));
+            var response = await _httpClient.SendAsync(message);
+            var stringResponse = await response.Content.ReadAsStringAsync();
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new HttpRequestException(stringResponse);
+            }
+
+            return JsonSerializer.Deserialize<Instance>(stringResponse, new JsonSerializerOptions{PropertyNameCaseInsensitive = true});
+        }
+
+        private string GetOrgToken(string id, string orgNumber)
+        {
+            // Copied from HomeController (should probably be a method of IAuthService)
+            List<Claim> claims = new List<Claim>();
+            string issuer = _generalSettings.Hostname;
+            claims.Add(new Claim(AltinnCoreClaimTypes.Org, id.ToLower(), ClaimValueTypes.String, issuer));
+            claims.Add(new Claim(AltinnCoreClaimTypes.AuthenticationLevel, "2", ClaimValueTypes.Integer32, issuer));
```

Comment:
Should be okay to set authentication level to level 3 here.

Reply:
Hmm... As the comment says, I just copied it from `HomeController.cs`.

Wouldn't it be required to be `4` in case someone sets 4 as a requirement in the `policy.xml` for their app? What level do you get for a maskinporten token? Is there a way to set `ObligationExpression` lower for org tokens?
![image](https://user-images.githubusercontent.com/131616/203353315-2965f4d3-2398-40ea-b311-b4e88d324efd.png)

Reply:
Authentication sets authentication level to 3 for Maskinporten during a token exchange into an Altinn token.

Reply:
It's possible to add a special rule for application owners.
https://docs.altinn.studio/app/development/configuration/authorization/rules/#org-can-access-with-authentication-level-3

Reply:
This is fixed in https://github.com/Altinn/altinn-studio/pull/8563/commits/e5aeacafa2982db96356a7462546e5ab7c5e5e32.

@SandGrainOne Would you mind taking another look?